cmd/bnsd/x/account/codec.proto

```diff
@@ -187,11 +203,13 @@ message RegisterAccountMsg {
   // leave the administration to the domain owner only.
   bytes owner = 4 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
   repeated BlockchainAddress targets = 5 [(gogoproto.nullable) = false];
-  // Third party token is an arbitrary data that can be provided by a middleman
+  // Broker is an arbitrary data that can be provided by a middleman
   // (i.e. retailer) that although does not participate directly in the
   // transaction, helped a user to renew a domain. Storing a third party token
-  // helps identify contribution of such party later.
-  bytes third_party_token = 6;
+  // helps identify contribution of such party later. Must be an email or
+  // a weave address that starts with a format or hex.
+  // example bech32:tiov16hzpmhecd65u993lasmexrdlkvhcxtlnf7f4ws
```

Comment:
Broker is a weave address (bech32 or hex) that can be provided by a middleman that helped facilitate the registration transaction.  For example, an IOV token holder that registers a domain in exchange for fiat from a client is a broker.  Storing the broker helps identify the contribution of such a party, which allows for automated commission distribution through an IOV reward initiative, for example.  Must be a weave address that starts with a format or hex, for example: bech32:tiov16hzpmhecd65u993lasmexrdlkvhcxtlnf7f4ws.